matrix_sdk_crypto/src/verification/requests.rs

```diff
@@ -17,8 +17,11 @@ use std::{
     time::Duration,
```

Comment:
The [`SUPPORTED_METHODS`](https://github.com/matrix-org/matrix-rust-sdk/blob/master/matrix_sdk_crypto/src/verification/requests.rs#L55) `const` needs to differ depending on if this feature is enabled, otherwise we'll advertise that we support QR code showing.